Bio/Graphics/GenomeDiagram/_Colors.py

```diff
@@ -30,6 +30,31 @@
 class ColorTranslator(object):
     """ Class providing methods for translating representations of color into
     """
+
+    """
```

Comment:
You need to remove the two triple quote lines above - right now your test is not part of the docstring, rather a self contained string in the function.